iree/compiler/Codegen/Passes.h

```diff
@@ -167,18 +167,18 @@ std::unique_ptr<OperationPass<FuncOp>> createLLVMCPUVectorizationPass(
 /// Replaces llvm.intr.fma with its unfused mul and add ops.
 std::unique_ptr<OperationPass<FuncOp>> createLLVMCPUUnfuseFMAOpsPass();
 
-/// A pass that converts vector dialect operations to inline assembly
-std::unique_ptr<OperationPass<FuncOp>>
-createVectorToAArch64InlineAssemblyPass();
+/// A pass that converts certain vector.contract ops to custom kernels using in
+/// either SIMD intrinsics or inline assembly
```

Comment:
Nit: typo

```suggestion
/// A pass that converts certain vector.contract ops to custom kernels using
/// either SIMD intrinsics or inline assembly
```

although actually, do we need to specify that it's intrinsics or inline asm? Might there be another reason for a custom lowering?